lib/cli/src/commands/publish.rs

```diff
@@ -0,0 +1,652 @@
+use anyhow::Context;
+use clap::Parser;
+use flate2::{write::GzEncoder, Compression};
+use rusqlite::{params, Connection, OpenFlags, TransactionBehavior};
+use std::fs;
+use std::io::{self, Write};
+use std::path::{Path, PathBuf};
+use tar::Builder;
+use thiserror::Error;
+use time::{self, OffsetDateTime};
+use wasmer_registry::publish::SignArchiveResult;
+use wasmer_registry::{PartialWapmConfig, PACKAGE_TOML_FALLBACK_NAME};
+
+const CURRENT_DATA_VERSION: i32 = 3;
```

Comment:
We should probably write a test to make sure `CURRENT_DATA_VERSION` is in sync with the number of available migrations.

Reply:
Done in https://github.com/wasmerio/wasmer/pull/3432/commits/e9ceff9e1620aa33c0f1327c0ddfb9af4440b3e5